src/marlin_stubs/pause/M600.cpp

```diff
@@ -160,11 +201,15 @@ void M600_manual() {
         parser.seen('U') ? std::make_optional(parser.value_axis_units(E_AXIS)) : std::nullopt,
         parser.seen('L') ? std::make_optional(parser.value_axis_units(E_AXIS)) : std::nullopt,
         parser.seen('E') ? std::make_optional(std::abs(parser.value_axis_units(E_AXIS))) : std::nullopt,
+        parser.seen('C') ? std::make_optional(filament::Colour::from_string(filamenttype)) : std::nullopt,
```

Comment:
Should it be `colourtype` instead of `filamenttype`?

Reply:
Indeed :)  Solved with commit [d631509](https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/3581/commits/d6315094ec575102dc032859100f6114a28a341d)